src/pages/index.tsx

```diff
@@ -1,436 +1,303 @@
-      description="APM description">
-        Monitor and analyze infrastructure health and app performance metrics.
-      </Translate>),
-    link: 'docs/apm',
-  },
-];
-
-const features4 = [
-  {
-    title: translate({
-      id: 'landing.feature.cse.title',
-      message: 'Cloud SIEM',
-      description: 'Title for CSE',
-    }),
-    imageUrl: 'img/icons/security/cloud-siem.png',
-    description: (<Translate
-      id="landing.feature.cse.desc"
-      description="CSE description">
-        Security event management and insight into key issues.
-      </Translate>),
-    link: 'docs/cse',
-  },
-  {
-    title: translate({
-      id: 'landing.feature.soar.title',
-      message: 'Cloud SOAR',
-      description: 'Title for SOAR',
-    }),
-    imageUrl: 'img/icons/security/SOC.png',
-    description: (<Translate
-      id="landing.feature.soar.desc"
-      description="SOAR description">
-        Modernize and automate your SOC for faster response times.
-      </Translate>),
-    link: 'https://www.sumologic.com/solutions/cloud-soar',
-  },
-];
-
-
-const features5 = [
-  {
-    title: translate({
-      id: 'landing.feature.alerts.title',
-      message: 'Alerts',
-      description: 'Title for alerts',
-    }),
-    imageUrl: 'img/icons/alerts.png',
-    description: (<Translate
-      id="landing.feature.alerts.desc"
-      description="alerts description">
-        Visualize your data and set alerts to monitor activity.
-      </Translate>),
-    link: 'docs/alerts',
-  },
-  {
-    title: translate({
-      id: 'landing.feature.dashboards.title',
-      message: 'Dashboards and Visuals',
-      description: 'Title for dashboards',
-    }),
-    imageUrl: 'img/icons/dashboards.png',
-    description: (<Translate
-        id="landing.feature.dashboards.desc"
-        description="Dashboards description">
-          Create visualizations, monitors, and alerts for your apps.
-        </Translate>),
-    link: 'docs/dashboards-new',
-  },
-  {
-    title: translate({
-      id: 'landing.feature.manage.title',
-      message: 'Manage Sumo',
-      description: 'Title for manage',
-    }),
-    imageUrl: 'img/icons/manage.png',
-    description: (<Translate
-        id="landing.feature.manage.desc"
-        description="Manage description">
-          Manage Sumo admin settings and advanced features.
-        </Translate>),
-    link: 'docs/manage',
-  },
-];
-
+      {/* Hero */}
+      <Stack
+        sx={{
+          bgcolor: 'white',
+          backgroundImage: `url(${bgImage})`,
+          backgroundPosition: {
+            md: 'top',
+            xs: 'left center',
+          },
+          backgroundRepeat: 'no-repeat',
+          backgroundSize: {
+            md: '100% 200%',
+            xs: '100% 100%',
+          },
+        }}
+        height={{
+          md: 'auto',
+          xs: '100%',
+        }}
+        py={4}
+        textAlign='center'
+      >
+        <Container maxWidth='lg'>
+          <Grid
```

Comment:
- Button font style doesn't quite match the original
- On hover, white border should disappear

Should look like this:
<img width="234" alt="Screen Shot 2022-12-01 at 2 25 55 PM" src="https://user-images.githubusercontent.com/56411016/205172668-ed41f1a1-2825-44e8-9e11-37b5e1cb2d76.png">

Reply:
What specifically about the font style?

https://user-images.githubusercontent.com/10472920/205726032-c8f2592b-eac2-4d07-8ccf-873198f1b19a.mov

Reply:
[Created follow-up issue for this](https://github.com/SumoLogic/sumologic-documentation/issues/1151)